crates/interfaces/src/test_utils/generators.rs

```diff
@@ -7,7 +7,7 @@ use reth_primitives::{
     SealedHeader, StorageEntry, Transaction, TransactionKind, TransactionSigned, TxLegacy, B256,
     U256,
 };
-use secp256k1::{KeyPair, Secp256k1};
+use secp256k1::{Keypair, Secp256k1};
```

Comment:
lol why

Reply:
Apparently mistake in the Rust library https://github.com/rust-bitcoin/rust-secp256k1/issues/647